container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_collector_functions/create_logs_collector.go

```diff
@@ -116,16 +116,16 @@ func CreateLogsCollectorForEnclave(
 		return nil, stacktrace.Propagate(err, "An error occurred getting logs collector object using container ID '%v', labels '%+v', status '%v' and host machine port bindings '%+v'", containerId, containerLabels, defaultContainerStatusForNewLogsCollectorContainer, hostMachinePortBindings)
 	}
 
-	logsCollectorAvailabilityChecker := fluentbit.NewFluentbitAvailabilityChecker(logsCollectorObj.GetBridgeNetworkIpAddress(), logsCollectorObj.GetPrivateHttpPort().GetNumber())
```

Comment:
`fluentbit.NewFluentbitAvailabilityChecker(logsCollectorObj.GetBridgeNetworkIpAddress(), logsCollectorObj.GetPrivateHttpPort().GetNumber())` was leaking implementation details and also didn't think there needed to be a specific fluent bit availability checker since its a simple http request so i made availability checker a general purpose function